### DISCUSSION TITLE
Confused about the xgb model

### DISCUSSION THREAD

**Discussion by marine1983boots** [kyleskom/NBA-Machine-Learning-Sports-Betting#148] [Q&A]
I need clarification with todays prediction. 

Detroit Pistons have an EV 0f 81.53

They are playing the Washington wizards, who are green on the selection. I assume the booster is telling me that it should be the wizards that will win? Why are the pistons high EV? How do I interpret this data?

![image](https://user-images.githubusercontent.com/127334214/224929514-5c86ab1a-ac44-461e-ad8e-a12654e50117.png)

**Comment by Marcfeitosa** [comment]
> I need clarification with todays prediction.
> 
> Detroit Pistons have an EV 0f 81.53
> 
> They are playing the Washington wizards, who are green on the selection. I assume the booster is telling me that it should be the wizards that will win? Why are the pistons high EV? How do I interpret this data?
> 
> ![image](https://user-images.githubusercontent.com/127334214/224929514-5c86ab1a-ac44-461e-ad8e-a12654e50117.png)

I also got this question

**Comment by Jc100047** [comment]
If the model gives a team a higher win probability than the oddsmakers implied probability then that bet would have a positive EV. 

To give an example, imagine you bet on a coin toss and the odds were +120 for heads (implied probability = 45.45%). In reality a coin toss has a 50% chance of landing on heads so in the long run you will eventually profit given enough bets if you only bet on heads. The idea is the same with EV, the higher the EV the higher the profitability of the bet.

In this case the model still predicts that the Wizards will win but it's more profitable to bet on the Pistons. In situations where the EV is negative for both teams you simply shouldn't bet.

**Reply by marine1983boots** [reply]
Ah got you. Makes sense. So for the example above and moving forward you should be betting the EV selection rather than the predicted winner?

So in this case in the long run betting pistons should be the choice.

**Reply by kyleskom** [reply]
Remember this should just be one tool in many used to evaluate a game.

**Reply by marine1983boots** [reply]
My bookie is bet365 in the code. When I have a positive EV. Where should I Place the bet? Bet365 or betfair for example. I've been looking into odds jam and they have multiple bookies. Just a little confused on how I apply the data provided.
